homeassistant/components/snips/__init__.py

```diff
@@ -150,23 +150,22 @@ def async_set_feedback(site_ids, state):
             )
             if "plain" in intent_response.speech:
                 snips_response = intent_response.speech["plain"]["speech"]
-        except intent.UnknownIntent:
-            _LOGGER.warning(
-                "Received unknown intent %s", request["intent"]["intentName"]
-            )
-        except intent.IntentError:
-            _LOGGER.exception("Error while handling intent: %s.", intent_type)
 
-        if snips_response:
-            notification = {
-                "sessionId": request.get("sessionId", "default"),
-                "text": snips_response,
-            }
+            notification = {"sessionId": request.get("sessionId", "default")}
+
+            if snips_response:
+                notification["text"] = snips_response
```

Comment:
Can we move this line into ‘if "plain" in ...‘?

Reply:
Yes, but notification should come above it.
Like so:

```
            notification = {"sessionId": request.get("sessionId", "default")}

            if "plain" in intent_response.speech:
                notification["text"] = intent_response.speech["plain"]["speech"]

```

Reply:
If agreed I can commit it.

Reply:
Agreed. Do we need snips_response at all? Maybe we can remove this dict completely?

Reply:
Yes, I already did :) 
Committing now and pushing